.github/workflows/build.yml

```diff
@@ -0,0 +1,77 @@
+name: Build
+
+on:
+  push:
+    paths-ignore:
+      - 'README.md'
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+
+jobs:
+  windows:
+    name: Windows
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        configuration: [Debug, Release]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Generate cmake files
+        shell: cmd
+        run: |
+          mkdir build && cd build
+          cmake .. -G "Visual Studio 16 2019" -A Win32 -D BUILD_TOOL=ON
+      - name: Build
+        run: |
+          cmake --build build/ --config ${{ matrix.configuration }}
+          Rename-Item -Path "build\${{ matrix.configuration }}\XbSymbolDatabaseTool.exe" -NewName "XbSymbolDatabaseTool.win64.${{ matrix.configuration }}.exe"
```

Comment:
`-A Win32` will output 32-bit executable on build, not 64-bit executable by the way.